NEWS.md

```diff
@@ -51,7 +51,7 @@ cvc5 1.2.0
   by cvc5. The plugin class also contains a callback for the user to introduce
   new learned clauses into the state of cvc5 during solving.
 
-- Added a new strategy `--mbqi-fast-sygus` (disabled by default) for **quantifier
+- Added a new strategy `--mbqi-enum` (disabled by default) for **quantifier
```

Comment:
We shouldn't retroactively change the release notes.  We will have to make a new item mentioning the name change under the latest release notes if you we decide to change the name.